task_10/toma.lobanova/src/main/java/com/hillel/tasks/BinarySearch.java

```diff
@@ -0,0 +1,23 @@
+package com.hillel.tasks;
+
+public class BinarySearch {
+
+  public boolean exists(int[] array, int number) {
+    return exists(array, number, 0, array.length - 1);
+  }
+
+  private boolean exists(int[] array, int number, int startIndex, int endIndex) {
+    if (endIndex < startIndex) {
+      return false;
+    }
+    int midIndex = startIndex + (endIndex - startIndex) / 2;
```

Comment:
yeah, I guess it is the same int midIndex = (endIndex + startIndex) / 2; I will correct